src/main/java/ttt/PlayerFactory.java

```diff
@@ -0,0 +1,43 @@
+package ttt;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class PlayerFactory {
+
+    public enum Marks {
+        X ("X"),
+        O ("O");
+
+        public final String mark;
+
+        Marks (String mark) {
+            this.mark = mark;
+        }
+    }
+
+    public static List<Player> create(int type) {
+        Player player1;
+        Player player2;
+        if (type == 1 || type == 2) {
```

Comment:
Having 1 and 2 here couples this class to the UI. You could use an enum (or something) for the menu id's, then refer to that enum here, creating a layer of abstraction.